src/components/05-utils/polyfill/README.md

```diff
@@ -0,0 +1,74 @@
+# AXA Patterns Library Polyfill
+
+This package includes polyfills for all AXA B2C Browsers. It includes 2 types of bundles:
+
+- Bundle for IE11 + WebComponents
+- Bundle for modern browser that don't support yet WebComponents.
```

Comment:
Correct: "Bundle for modern browser that don't support WebComponents yet."

Reply:
Done